generators/server/templates/_pom.xml

```diff
@@ -1203,37 +1203,12 @@
             </dependencies>
             <build>
                 <plugins>
-                    <%_ if (!skipClient) { _%>
+                    <%_ if (!skipClient && clientFw === 'angular2') { _%>
```

Comment:
Why do we have this new `clientFw` variable? There is already an `angularVersion` variable